doc/src/modules/numeric-computation.rst

```diff
@@ -1,3 +1,543 @@
+===============
+Code Generation
+===============
```

Comment:
I asked @tokencolour to combine his new document into the numeric computation page but to keep the title "Numeric Computation". It is no longer in the tutorial. I'm all ears for any suggestions on how to organize the content though.